[BUG] RDSTRegressor expected results on covid3month

### Describe the bug

The results produced locally do not match those produced in the CI for RDSTRegressor on Covid3Month. The expected results seem to be fine for the other dataset and for RDSTClassifier.

### Steps/Code to reproduce the bug

```python
from aeon.testing.expected_results.results_reproduction.regressor_results_reproduction import _print_results_for_regressor
_print_results_for_regressor("RDSTRegressor", "Covid3Month")
```
Outputs :
```python
RDSTRegressor - Covid3Month
[
0.0687,
0.071,
0.0227,
0.0425,
0.0834,
0.0251,
0.0064,
0.1398,
0.0667,
0.031,
]
```

### Expected results

The test do not fail and results produced locally match those produce in the CI

### Actual results

```python-traceback
FAILED aeon/testing/tests/test_all_estimators.py::test_all_estimators[check_regressor_against_expected_results(estimator_class=RDSTRegressor)] - AssertionError: 
Arrays are not almost equal to 2 decimals
Failed to reproduce results for RDSTRegressor on Covid3Month
Mismatched elements: 6 / 10 (60%)
Max absolute difference: 0.10256623
Max relative difference: 4.7437444
 x: array([0.04, 0.04, 0.04, 0.04, 0.04, 0.04, 0.04, 0.04, 0.04, 0.04])
 y: array([0.07, 0.07, 0.02, 0.04, 0.08, 0.03, 0.01, 0.14, 0.07, 0.03])
```

### Versions

System:
    python: 3.10.13 | packaged by Anaconda, Inc. | (main, Sep 11 2023, 13:24:38) [MSC v.1916 64 bit (AMD64)]
executable: C:\Users\antoine\anaconda3\envs\aeon_dev\python.exe
   machine: Windows-10-10.0.22631-SP0
Python dependencies:
         aeon: 0.11.1
          pip: 23.3.1
   setuptools: 68.0.0
 scikit-learn: 1.2.2
        numpy: 1.24.4
        numba: 0.58.1
        scipy: 1.11.4
       pandas: 2.0.3

--- a/aeon/testing/testing_config.py
+++ b/aeon/testing/testing_config.py
@@ -55,7 +55,6 @@ EXCLUDED_TESTS = {
     "ClaSPSegmenter": ["check_non_state_changing_method"],
     "HMMSegmenter": ["check_non_state_changing_method"],
     # Unknown issue not producing the same results
-    "RDSTRegressor": ["check_regressor_against_expected_results"],
     "RISTRegressor": ["check_regressor_against_expected_results"],
     # Requires y to be passed in inverse_transform,
     # but this is not currently enabled/supported

